test/extended/operators/olm.go

```diff
@@ -251,50 +251,30 @@ var _ = g.Describe("[Feature:Platform] an end user use OLM", func() {
 		}
 	})
 
-	// OCP-27589 do not use ipv4 addresses in CatalogSources generated by marketplace
 	// author: tbuskey@redhat.com
-	g.It("do not use ipv4 addresses in CatalogSources generated by marketplace", func() {
-		re := regexp.MustCompile(`(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}`)
-		olmErrs := 0
+	g.It("OLM-Low-OCP-24058-components should have resource limits defined", func() {
```

Comment:
The importance of this [OCP-24058](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-24058) is `Medium`, not `Low`. Please update it, thanks!